ast/parser.go

```diff
@@ -330,9 +330,14 @@ func (p *Parser) Parse() ([]Statement, []*Comment, Errors) {
 		s = p.save()
 
 		if imp := p.parseImport(); imp != nil {
+			if RegoRootDocument.Equal(imp.Path.Value.(Ref)[0]) {
+				p.regoV1Import(imp, allowedFutureKeywords)
```

Comment:
I wonder, should we disregard the `future_keywords` capability when `rego.v1` is imported? 🤔 . I'm leaning towards "yes".

We should probably also remove this capability parameter in `1.0`. Didn't think of that before.